### DISCUSSION TITLE
Should charge oscillate between the lower and upper limits automatically?

### DISCUSSION THREAD

**Discussion by ghost** [mhaeuser/Battery-Toolkit#63] [Q&A] [answered]
Not sure if I'm misunderstanding intended behavior. 
I gathered from the readme that this app's automated sequence of events should be: charge up to the upper limit, then disable the power input and discharge down to the lower limit, then re-enable the power input and repeat. 

It appears on my M1 Pro that upon reaching the upper limit it just holds there, and I have to manually press "Disable Power Adapter" to get it to start discharging.

**Comment by mhaeuser** [comment, marked as answer]
Works as intended. The readme says battery charging is disabled, not power as a whole. The lower limit is for “creep discharging”.
Related: https://github.com/mhaeuser/Battery-Toolkit/discussions/30

**Reply by ghost** [reply]
Ok thank you!
